llvm/test/CodeGen/AMDGPU/promote-alloca-invariant-markers.ll

```diff
@@ -1,25 +1,75 @@
-; RUN:  llc -amdgpu-scalarize-global-loads=false  -mtriple=amdgcn-- -mattr=+promote-alloca -verify-machineinstrs < %s | FileCheck -check-prefix=GCN %s
-target datalayout = "A5"
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
```

Comment:
I move all contents from `llvm/test/CodeGen/AMDGPU/promote-alloca-invariant-marks.ll` to this file and remove all existing contents as they are no longer useful.